website/src/pages/docs/comparison.mdx

```diff
@@ -6,6 +6,13 @@ import { Callout } from '@theguild/components'
 
 This comparison highlights the main differences when using Yoga instead of Apollo Server.
 
+### Better runtime performance
+
+GraphQL Yoga has significantly less latency and much higher requests rate than Apollo Server in all [popular benchmarks](https://github.com/graphql-crystal/benchmarks).
+including when using Apollo Federation - see benchmarks [here](https://github.com/ardatan/benchmarks/tree/federation#apollo-federation-server-benchmarks)
+<img width="742" alt="image" src="https://user-images.githubusercontent.com/1297448/222475526-7fa4b418-e83e-4201-956d-8e82ecf5c909.png">
```

Comment:
I think it's worth moving the file to the repository. It may be deleted at some point.

Reply:
Or better yet, just embed the table.

```suggestion
| Name                          | Language      | Server          | Latency avg      | Requests      |
| ----------------------------  | ------------- | --------------- | ---------------- | ------------- |
| [GraphQL Yoga with Response Cache](https://github.com/dotansimha/graphql-yoga) | Node.js | http | 46.54ms | 2.2kps |
| [GraphQL Yoga with JIT](https://github.com/dotansimha/graphql-yoga) | Node.js | http | 764.83ms | 120ps |
| [GraphQL Yoga](https://github.com/dotansimha/graphql-yoga) | Node.js | http | 916.90ms | 100ps |
| [Apollo Server](https://github.com/apollographql/apollo-server) | Node.js | Express | 1,234.12ms | 64ps |
```

Reply:
+ maybe we should add the `@cacheControl` (Apollo Server) to have 1-to-1 comparison with `GraphQL Yoga with Response Cache`.